### ISSUE TITLE
A little issue about RPN box loss

### ISSUE THREAD

**Issue by jacktkk** [ruotianluo/pytorch-faster-rcnn#91]
I wonder why u calculate the RPN class loss just use 256 example bboxes with 1:1（pos : neg）
But for bbox loss, u consider all the bbox that generated by def generate_anchors_pre? why don't u choose the same 256 example boxes like RPN class loss?

thx for your reply!
@ruotianluo

**Comment by ruotianluo** [comment]
https://github.com/ruotianluo/pytorch-faster-rcnn/blob/master/lib/layer_utils/anchor_target_layer.py#L109

Because apparently we want to have more training data; however the positive and negative are still balanced.
